src/SprykerSdk/Zed/AppSdk/Communication/Console/AppTranslationBuilderConsole.php

```diff
@@ -0,0 +1,142 @@
+     *
+     * @return int
+     */
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $appTranslationRequestTransfer = new AppTranslationRequestTransfer();
+
+        $appTranslationRequestTransfer->setConfigurationFile($input->getOption(static::TRANSLATION_FILE));
+
+        $this->getTranslationsInput($input, $output);
+
+        $appTranslationRequestTransfer->setTranslations($this->translations);
+
+        $appConfigurationResponseTransfer = $this->getFacade()->appTranslationCreate($appTranslationRequestTransfer);
+
+        if ($appConfigurationResponseTransfer->getErrors()->count() === 0) {
+            return static::CODE_SUCCESS;
+        }
+
+        // @codeCoverageIgnoreStart
+        if ($output->isVerbose()) {
+            foreach ($appConfigurationResponseTransfer->getErrors() as $error) {
+                $output->writeln($error->getMessageOrFail());
+            }
+        }
+
+        return static::CODE_ERROR;
+        // @codeCoverageIgnoreEnd
+    }
+
+    /**
+     * @param \Symfony\Component\Console\Input\InputInterface $input
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     * @param string $questionText
+     *
+     * @return string
+     */
+    protected function askTextQuestion(InputInterface $input, OutputInterface $output, string $questionText): string
+    {
+        return $this->getHelper('question')->ask($input, $output, new Question($questionText));
+    }
+
+    /**
+     * @param \Symfony\Component\Console\Input\InputInterface $input
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     *
+     * @return void
+     */
+    protected function getTranslationsInput(InputInterface $input, OutputInterface $output): void
+    {
+        do {
+            $translationKey = $this->askTextQuestion($input, $output, 'Please enter a translation key: ');
+
+            $this->setLocale(
+                $input,
+                $output,
+                $translationKey,
+            );
+        } while ($this->askForConfirmation($input, $output, 'Do you want to add more translation?') == 'Yes');
+    }
+
+    /**
+     * @param \Symfony\Component\Console\Input\InputInterface $input
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     * @param string $translationKey
+     *
+     * @return void
+     */
+    protected function setLocale(InputInterface $input, OutputInterface $output, string $translationKey): void
+    {
+        do {
+            $localeName = $this->askTextQuestion($input, $output, 'Please enter Locale name: ');
+            $translationValue = $this->askTextQuestion($input, $output, 'Please enter translation value: ');
+
+            $this->translations[$translationKey][$localeName] = $translationValue;
+        } while ($this->askForConfirmation($input, $output, 'Do you want to add more Locale?') == 'Yes');
```

Comment:
Sounds weird to me. Should it be `Do you want to add more locales?`

Reply:
I changed it as suggested.